src/constants/constants.js

```diff
@@ -0,0 +1,2 @@
+export const CalednlyClarityCallLink =
```

Comment:
```suggestion
export const CalendlyClarityCallLink =
```
Sorry typo in my original comment.  Also prettier is failing.